mfr/providers/osf/provider.py

```diff
@@ -96,6 +99,12 @@ def __init__(self, request, url):
         # }}
 
         name, ext = os.path.splitext(metadata['data']['name'])
+
+        if MAX_FILE_SIZE_TO_RENDER.get(ext) and metadata['data']['size'] > MAX_FILE_SIZE_TO_RENDER.get(ext):
+            size = sizeof_fmt(MAX_FILE_SIZE_TO_RENDER.get(ext))
+            raise TooBigToRenderError("This file with extension '{ext}' exceeds the size limit of {size} and will not "
+                                      "be rendered. To view this file download it and view it offline.".format(**locals()))
```

Comment:
I'd prefer this to be explicit, either with `.format(ext=ext, size=size)` or positionals.  I'm squeamish about possibly exposing local private variables in an error message.